bin/main.rb

```diff
@@ -1,2 +1,74 @@
 #!/usr/bin/env ruby
-puts 'hello, world'
+class Game
+  def initialize
+    @players = Players.new
+  end
+
+  def move
+    @end = false
+    @gameboard = Board.new
+    @turn = 1
+    while @turn < 10
+      if @turn.odd?
+        turn_sequence(@players.player1)
+      elsif @turn.even?
+        turn_sequence(@players.player2)
+      end
```

Comment:
```while @turn < 10``` this condition is not right, because the game may finish earlier

Make ```@end``` become true when its a wining move or a draw and break the loop